src/dom/dom.js

```diff
@@ -1059,12 +1061,12 @@ p5.prototype.createInput = function(value = '', type = 'text') {
 };
 
 /**
- * Creates an `&lt;input&gt;&lt;/input&gt;` element in the DOM of type 'file'.
+ * Creates an `&lt;input&gt;&lt;/input&gt;` element in the DOM of type `'file'`.
  * This allows users to select local files for use in a sketch.
  *
  * @method createFileInput
- * @param  {Function} callback callback function for when a file is loaded
- * @param  {Boolean} [multiple] optional, to allow multiple files to be selected
+ * @param  {Function} callback Callback function to run when a file is loaded
+ * @param  {Boolean} [multiple] (optional) Allow multiple files to be selected
```

Comment:
"(optional)" is not required here as it is added automatically when the parameter is marked as so.